README.md

```diff
@@ -43,17 +59,9 @@ pnpm add solid-styled babel-plugin-solid-styled
 }
 ```
 
-### Typescript
-
-Add this to any d.ts file
-
-```ts
-/// <reference types="solid-styled" />
-```
-
 ### `<StyleRegistry>`
 
-`<StyleRegistry>` manages the lifecycle of stylesheets. It is required only to be used once, preferably at the root of your SolidJS application
+`<StyleRegistry>` manages the lifecycle of stylesheets (for instance styles of a component get removed from memory if all instances of that component unmount). It needs to be included only once, preferably at the root of your SolidJS application.
```

Comment:
It's probably helpful to mention that this only happens on client-side.

Reply:
Done